### PR TITLE
Improve swaylock man

### LINKED ISSUE #1018: Swaylock does not put colors over an image
Using:

```
swaylock -c 00000044 -i /usr/share/backgrounds/f25/default/normalish/f25.png -s
```

I was expecting to see the image with a "darker" tone. I do see the image in a "plain" way.  In the doc it says "If an image is in use, also enables color display.", so I guess this is a bug

### PR DESCRIPTION
**Pull request by Fale**
Fixes #1018

### CODE REVIEW

**Comment by SirCmpwn** [comment]
Thanks!

File: swaylock/swaylock.1.txt [hunk 1]
```diff
@@ -24,10 +24,11 @@ Options
 
 *-c, \--color* <rrggbb[aa]>::
 	Turn the screen into the given color instead of white. If an image is in use,
-	also enables color display.
+	also enables color display. This option and -i are mutually exclusive.
 
 *-i, \--image* [<output>:]<path>::
-	Display the given image, optionally only on the given output.
+	Display the given image, optionally only on the given output. This option
+	and -i are mutually exclusive.
```
Comment (jhalmen) [inline swaylock/swaylock.1.txt]: shouldn't `-i` here be `-c` ?

Comment (Fale) [inline swaylock/swaylock.1.txt]: yes, and @SirCmpwn already fixed it in master

**Comment by jhalmen** [comment]
should've seen that. thanks

**Pull request merged**